src/Microsoft.AspNetCore.Blazor/Routing/NavLink.cs

```diff
@@ -73,6 +73,7 @@ public void SetParameters(ParameterCollection parameters)
             _renderHandle.Render(Render);
         }
 
+        /// <inheritdoc />
```

Comment:
Note that this doesn't actually work (I don't think). Our docs stuff doesn't actually honour this